src/main/java/ee/tuleva/onboarding/auth/ocsp/CheckCertificateResponse.java

```diff
@@ -11,9 +12,14 @@
 @ToString
 public class CheckCertificateResponse implements Serializable, Person {
 
-  private static final long serialVersionUID = 333351175769353073L;
+  @Serial private static final long serialVersionUID = 333351175769353073L;
 
   private final String firstName;
   private final String lastName;
   private final String personalCode;
+
+  @Override
+  public String getPhoneNumber() {
+    return null;
+  }
```

Comment:
We should not have the phoneNumber as part of the top-level interface if it's not relevant for all the implementation classes.